File: abc/abc189/abc189_a/abc189_a.cs
using System;

public class A
{
  static void Main()
  {
    string s = Console.ReadLine();
    for(int i = 0; i < 3; i++)
    {
      for(int j = i; j < 3; j++)
      {
        if(s[i] != s[j])
        {
          Console.Write("Lost");
          return;
        }
      }
    }

    Console.Write("Won");
  }
}

File: abc/abc189/abc189_a/abc189_a.rs
use proconio::input;

fn main() {
  input! {
    s: String,
  }

  for c in s.chars() {
    if c != s.chars().nth(0).unwrap() {
      print!("Lost");
      return;
    }
  }

  print!("Won");
}
